### PR TITLE
Performance guide

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-performance-guidelines.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
+[id='{context}-performance-guidelines']
+= Performance Guidelines
+
+{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that aren't part of the standard applications.
+
+Node.js based server can rely on customer prefered database or external api and any other datasource. This  may have significant impact on overal application performance. {PROJECT_NAME} was desigined to expose areas and configurations that can have .
```
Comment (paolobueno) [inline docs/workforce-management-framework/topics/con-performance-guidelines.adoc]: Typo for 'preferred', capitalize 'API', double spacing.

Also seems like the sentence is incomplete here.

File: docs/workforce-management-framework/topics/con-performance-guidelines.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+== Server application
+
+Server exposes number of parameters that can be changed by adjusting server configuration.
+For more information see configuration
+link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-prod.js[config-prod.js]
+
+=== MongoDB
+
+MongoDB connection options should be configured to properly maintain conection pooling. See server side configuration.
+link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-prod.js[config-prod.js]
```
Comment (paolobueno) [inline docs/workforce-management-framework/topics/con-performance-guidelines.adoc]: Maybe add a link to official mongodb documentation around what options could be tweaked.

File: docs/workforce-management-framework/topics/con-performance-guidelines.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- assignee
+
+Workflow
+
+- id
+
+=== Redis Cache
+
+{PROJECT_NAME} using redis indirectly to store user sessions.
+Please refer to official redis client options.
```
Comment (paolobueno) [inline docs/workforce-management-framework/topics/con-performance-guidelines.adoc:41]: here also a link to the redis client docs

File: docs/workforce-management-framework/topics/con-performance-guidelines.adoc (hunk 2 above)
Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-performance-guidelines.adoc]: Had that before, but their version may differ. 
Going to include docs for the latest

Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-performance-guidelines.adoc]: Done

File: docs/workforce-management-framework/topics/con-performance-guidelines.adoc [hunk 4]
```diff
@@ -1,9 +1,9 @@
 [id='{context}-performance-guidelines']
 = Performance Guidelines
 
-{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that aren't part of the standard applications.
+{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that are not part of the standard applications. Node.js based server can rely on customer preffered database or external API. This may have significant impact on overal application performance.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-performance-guidelines.adoc]: Add an 'a' before customer, spelling  `preferred`
```Node.js based server can rely on a customer preferred database or external API.```

File: docs/workforce-management-framework/topics/con-performance-guidelines.adoc [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 
-Node.js based server can rely on customer preffered database or external API. This  may have significant impact on overal application performance. {PROJECT_NAME} was desigined to expose all configuration into single files.
+{PROJECT_NAME} was desigined to expose all configuration into single files.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-performance-guidelines.adoc]: Spelling ```designed ```
{PROJECT_NAME} was designed to expose all configuration into single files.

File: docs/workforce-management-framework/topics/con-production-checklist.adoc [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
+Server application provides abstraction for configuration that allows users to create profiles for different environments
 Application is created with 2 default profiles:
 
 - development (link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-dev.js[config-dev.js])
 - production (link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-prod.js[config-prod.js])
 
 For more information about profiles go to link:{context}-ref-server[Server side reference]
 
-Whem moving to production developers should review their configuration to make sure that a
+Whem moving to production developers should review server configuration.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-production-checklist.adoc]: Spelling ```When```
When moving to production developers should review server configuration.

File: docs/workforce-management-framework/topics/con-production-checklist.adoc [hunk 7]
```diff
@@ -51,8 +50,8 @@ Most important values
 
 Apache Cordova configuration needs to be adjusted before doing release.
 Configuration can be found in link:{WFM-RC-MobileURL}{WFM-RC-Release-Tag}/src/config.xml[Cordova config.xml file]
-Developers should review cordova file by removing permissions and plugins that aren't used.
-Additionally developers should restrict and remove any origins that aren't used in the application
+Developers should review cordova file by removing permissions and plugins that are not used.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-production-checklist.adoc]: caps ```Cordova``` 
Developers should review Cordova file by removing permissions and plugins that are not used.

**Review by austincunningham — CHANGES_REQUESTED** [review]

**Review by austincunningham — APPROVED** [review]

**Pull request merged**